### ISSUE TITLE
Upgrade from 0.2.1 to 0.2.2 fails

### ISSUE THREAD

**Issue by sobolevn** [peterjc/flake8-rst-docstrings#38]
CI link: https://github.com/wemake-services/wemake-python-styleguide/pull/2003/checks?check_run_id=2488908765

Error message:

```
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/multiprocessing/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/multiprocessing/pool.py", line 44, in mapstar
    return list(map(*args))
  File "/home/runner/work/wemake-python-styleguide/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/checker.py", line 676, in _run_checks
    return checker.run_checks()
  File "/home/runner/work/wemake-python-styleguide/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/checker.py", line 589, in run_checks
    self.run_ast_checks()
  File "/home/runner/work/wemake-python-styleguide/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/checker.py", line 494, in run_ast_checks
    for (line_number, offset, text, _) in runner:
  File "/home/runner/work/wemake-python-styleguide/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8_rst_docstrings.py", line 193, in run
    docstring,
AssertionError: Bad start line, node line number 1 for: See :term:`preset` in the docs.

"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/runner/work/wemake-python-styleguide/wemake-python-styleguide/.venv/bin/flake8", line 8, in <module>
    sys.exit(main())
  File "/home/runner/work/wemake-python-styleguide/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/main/cli.py", line 22, in main
    app.run(argv)
  File "/home/runner/work/wemake-python-styleguide/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/main/application.py", line 363, in run
    self._run(argv)
  File "/home/runner/work/wemake-python-styleguide/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/main/application.py", line 351, in _run
    self.run_checks()
  File "/home/runner/work/wemake-python-styleguide/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/main/application.py", line 264, in run_checks
    self.file_checker_manager.run()
  File "/home/runner/work/wemake-python-styleguide/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/checker.py", line 321, in run
    self.run_parallel()
  File "/home/runner/work/wemake-python-styleguide/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/checker.py", line 287, in run_parallel
    for ret in pool_map:
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/multiprocessing/pool.py", line 354, in <genexpr>
    return (item for chunk in result for item in chunk)
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/multiprocessing/pool.py", line 748, in next
    raise value
AssertionError: Bad start line, node line number 1 for: See :term:`preset` in the docs.

```

**Comment by peterjc** [comment]
I can drop the assert as a quick short term fix, but it would be great if you can point me at the failing example for testing purposes please.

**Comment by peterjc** [comment]
Found it, a one line ``__init__.py`` file:

```python
"""See :term:`preset` in the docs."""
```

https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/presets/__init__.py

**Comment by peterjc** [comment]
Should be fixed in v0.2.3
